tools.info

```diff
@@ -0,0 +1,8 @@
+GCC_VERSION="gcc7.3.0-OEL6.4+1.1"
+MSVC_VERSION="VS2017-15.5.5+1.1"
```

Comment:
Do you want to prefix platform-specific tools (e.g. WIN_... ) versus generic tools (e.g. gradle) ?

Reply:
I thought it would be redundant. If you think it is good to have then I don't have any objection.

Reply:
I don't have a strong opinion one way or the other.

Reply:
I agree it is redundant, but it helps developers who want to get a list of requirements. However, this might be better on the wiki then. Hence, I'm ok skipping the prefix.

Reply:
That is, I'm ok with the current proposal. (which has prefixes for the toolchain)